portal-api/tsconfig.json

```diff
@@ -1,28 +1,28 @@
 {
     "compilerOptions": {
       /* Visit https://aka.ms/tsconfig.json to read more about this file */
-  
+
       /* Projects */
       // "incremental": true,                              /* Enable incremental compilation */
       // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
       // "tsBuildInfoFile": "./",                          /* Specify the folder for .tsbuildinfo incremental compilation files. */
       // "disableSourceOfProjectReferenceRedirect": true,  /* Disable preferring source files instead of declaration files when referencing composite projects */
       // "disableSolutionSearching": true,                 /* Opt a project out of multi-project reference checking when editing. */
       // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
-  
+
       /* Language and Environment */
       "target": "ES6",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
       "lib": ["dom", "esnext"],                                      /* Specify a set of bundled library declaration files that describe the target runtime environment. */
       // "jsx": "preserve",                                /* Specify what JSX code is generated. */
-      // "experimentalDecorators": true,                   /* Enable experimental support for TC39 stage 2 draft decorators. */
+      "experimentalDecorators": true,                   /* Enable experimental support for TC39 stage 2 draft decorators. */
       // "emitDecoratorMetadata": true,                    /* Emit design-type metadata for decorated declarations in source files. */
```

Comment:
I think this option also need to be set to `true` (see the `tsconfig.json` file [here](https://github.com/UK-Export-Finance/dtfs2/pull/2684/files#diff-5b159449142f44f7a41c4d51121444bc51364a3331a07405bb0588c60531048e))

Reply:
Ah whoops I hadn't pushed that commit yet 😅